crates/biome_js_analyze/tests/specs/a11y/useSemanticElements/invalid.jsx

```diff
@@ -35,10 +35,9 @@
     <div role="caption" ></div>
     <div role="main" ></div>
     <div role="time" ></div>
-    <div role="p" ></div>
-    <div role="aside" ></div>
+    <div role="paragraph" ></div>
+    <div role="complementary" ></div>
     <div role="blockquote" ></div>
-    <div role="associationlist" ></div>
```

Comment:
The role was part of the Aria 1.2 draft and was eventually removed before the stabilization of Aria 1.2.